src/components/Checkbox.js

```diff
@@ -47,77 +47,77 @@ const defaultProps = {
     onMouseDown: undefined,
 };
 
-class Checkbox extends React.Component {
-    constructor(props) {
-        super(props);
+const Checkbox = (props) => {
+    const [isFocused, setIsFocused] = useState(false);
 
-        this.handleSpaceKey = this.handleSpaceKey.bind(this);
-        this.firePressHandlerOnClick = this.firePressHandlerOnClick.bind(this);
-    }
-
-    handleSpaceKey(event) {
+    const handleSpaceKey = (event) => {
```

Comment:
asked in the [slack](https://expensify.slack.com/archives/C01GTK53T8Q/p1686536725768839?thread_ts=1685727880.152119&cid=C01GTK53T8Q) open-source channel if we have preference for arrow function vs. the `function` keyword for functions defined inside components

Reply:
Cool, from the linter looks like we should use `const` here since we use those as props